man/systemd.exec.xml

```diff
@@ -89,6 +89,17 @@
 
     <variablelist class='unit-directives'>
 
+      <varlistentry>
+        <term><varname>BinaryDirectory=</varname></term>
+
+        <listitem><para>Takes a directory path relative to which the executable used by
+        the Exec*= (e.g. ExecStart=, ExecStop=, etc.) properties can be found.
+        The name of the executable has to be nonstandard.
```

Comment:
This is not well specified: I guess "nonstandard" means "not already existing in PATH", but you should just say that if that's what you mean :-)